### DISCUSSION TITLE
Casual chat with Max

### DISCUSSION THREAD

**Discussion by emmanuelbernard** [quarkusio/quarkus#40439] [Community]
Hey Max how are you doing as we explore Discussions?

**Comment by maxandersen** [comment]
hey - so quiet here ;)

**Comment by emmanuelbernard** [comment]
Oh no threading model :/ Why people don't get the good stuff form emails? ;(

**Reply by dmlloyd** [reply]
There is *sort of* a threading model...

**Reply by dmlloyd** [reply]
In the "slack was such a great idea!" sense

**Comment by maxandersen** [comment]
On Fri, May 3, 2024 at 2:44 PM Emmanuel Bernard ***@***.***>
wrote:

> Oh no threading model :/ Why people don't get the good stuff form emails?
> ;(
>

Well i can at least quote you?

**Comment by maxandersen** [comment]
> In the "slack was such a great idea!" sense
>

just checking this level of threading works  via email :P

**Reply by dmlloyd** [reply]
Looks like not!

**Reply by maxandersen** [reply]
correct - well that sucks ;/

**Reply by maxandersen** [reply]
reported here: https://github.com/orgs/community/discussions/3287
